src/web/src/app/space/detail/component.html

```diff
@@ -0,0 +1,51 @@
+<h1>{{title}}</h1>
+	<hr class="width-and-clear">
+	<div class="space-invites-heading">
+		<h2>Приглашения</h2>
+		<button class="space-invites-heading-addbtn mat-icon-button">+</button>
```

Comment:
А что делает space-invites-heading-addbtn?
```suggestion
		<button mat-icon-button><mat-icon>person_add</mat-icon></button>
```

Reply:
Готово, поправил.